From f939d3e07e1175caac5cf4cbf54bd8a58c2f198a Mon Sep 17 00:00:00 2001
From: Alex Kiernan <alex.kiernan@gmail.com>
Date: Fri, 7 Aug 2020 15:30:52 +0000
Subject: [PATCH] Fix field 'efi_loader_entry_one_shot_stat' has incomplete
 type

| In file included from ../../../../../../workspace/sources/systemd/src/login/logind-gperf.gperf:7:
| ../../../../../../workspace/sources/systemd/src/login/logind.h:131:21: error: field 'efi_loader_entry_one_shot_stat' has incomplete type
|   131 |         struct stat efi_loader_entry_one_shot_stat;
|       |                     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Upstream-Status: Inappropriate [musl specific]
Signed-off-by: Alex Kiernan <alex.kiernan@gmail.com>
---
 src/login/logind.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/login/logind.h b/src/login/logind.h
index e64ecce8e2..2d4007d84b 100644
--- a/src/login/logind.h
+++ b/src/login/logind.h
@@ -2,6 +2,9 @@
 #pragma once
 
 #include <stdbool.h>
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <unistd.h>
 
 #include "sd-bus.h"
 #include "sd-device.h"
